test/verify/machineslib.py

```diff
@@ -1133,8 +1133,11 @@ def installWithErrorTest(dialog):
         # to start properly.
         # This is applicable only for the following test so let's keep it last,
         # in order to allow the rest of the tests to run faster with QEMU KVM
-        # Run modprobe -r in retry loop because it fails sometimes with 'Module kvm is in use'
-        self.machine.execute("for i in 1 2 3 4 5; do modprobe -r kvm_intel && modprobe -r kvm && break || sleep 1; done")
+        try:
+            self.machine.execute("modprobe -r kvm_intel && modprobe -r kvm")
```

Comment:
You should probably include "kvm_amd" as you cannot assume that everyone who is developing cockpit uses only intel CPUs.

Reply:
Good point. Thanks